tx-source/site_core.php

```diff
@@ -1930,3 +1930,9 @@
 // Quotes/Formats
 $Definition['Toggle Quote'] = 'Toggle Quote';
 $Definition['Spoiler Warning'] = 'Spoiler Warning';
+
+// Catalogue Display Plugin/Formats
+$Definition['You must have the Post and Mobile Formats set to anything but "Text" in the Advanced Editor Plugin.'] = 'You must have the Post and Mobile Formats set to anything but "Text" in the Advanced Editor Plugin.';
+$Definition['Each discussion will show an uploaded image on the Discussions page instead of the author information. This only applies to categories with "Discussions" as the "Display As.'] = 'Each discussion will show an uploaded image on the Discussions page instead of the author information. This only applies to categories with "Discussions" as the "Display As.';
+$Definition['Default placeholder'] = 'Default placeholder';
+$Definition['Uplaod a placeholder image that will display when users do not have an image in their post.'] = 'Uplaod a placeholder image that will display when users do not have an image in their post.';
```

Comment:
```suggestion
$Definition['Upload a placeholder image that will display when users do not have an image in their post.'] = 'Upload a placeholder image that will display when users do not have an image in their post.';
```

Typo here. You will probably need to update over in the plugin as well.